client/room.html

```diff
@@ -216,118 +215,312 @@ <h1>Players</h1>
+          <p>This button will take the current state of this room and it as a new variant.</p>
+          <button icon="save">Save room state</button>
+        </div>
+        <div icon="upload">
+          <p>If you have a VTT or PCIO file, you can add their variants as new variants here.</p>
+          <button icon="upload">Upload files</button>
+        </div>
+        <div icon="link">
+          <p>You can add a variant from the public library to your game in case you want to combine different games into one or if you want to change the game metadata.</p>
+          <select></select>
+          <button icon="link">Link to public library</button>
+        </div>
+      </div>
     </div>
 
-    <div id="stateEditOverlay" class="overlay">
-      <h1>Edit game</h1>
-      <div class="details">
-        <div class="input"><label for="stateName">Name:</label> <input id="stateName"/></div>
-        <div class="input"><label for="stateYear">Year:</label> <input id="stateYear"/></div>
-        <div class="input"><label for="stateRules">Rules link:</label> <input id="stateRules"/></div>
-        <div class="input"><label for="stateMode">Mode:</label> <input id="stateMode"/></div>
-        <div class="input"><label for="stateTime">Time:</label> <input id="stateTime"/></div>
-        <div class="input"><label for="stateBGG">BoardGameGeek link:</label> <input id="stateBGG"/></div>
-        <div class="input"><label for="stateImage">Image URL:</label> <input id="stateImage"/><button class="ui-button uploadAsset">Upload</button></div>
-        <div class="input"><label for="stateLink">Update link:</label> <input id="stateLink"/></div>
-      </div>
-      <h2>Variants</h2>
-      <div id="variantsEditList">
-        <template id="template-variantseditlist-entry" type="text/template">
-          <div class="input"><label>Language (country code):</label> <input class="stateLanguage"/></div>
-          <div class="input"><label>Players:</label> <input class="statePlayers"/></div>
-          <div class="input"><label>Variant:</label> <input class="stateVariant"/></div>
-          <div class="input"><label>Attribution:</label> <textarea class="stateAttribution"></textarea></div>
-          <div class="input"><label>Update link:</label> <input class="stateLink"/></div>
-          <div class="editOverlayButtonBar">
-            <button class="ui-button green update update-current-state">Update variant using current room state</button>
-            <button class="ui-button download download-variant">Download variant</button>
-            <button class="ui-button red remove remove-variant">Remove variant</button>
+    <div id="stateDetailsOverlay" class="overlay notEditing">
+      <div class="buttons one">
+        <button icon="close" class="hideForEdit" id="closeDetails"></button>
+        <button icon="close" class="onlyForEdit red" id="discardDetails">Discard</button>
```

Comment:
maybe this should be cancel. I was editing the game and I wanted to cancel the edits. It took me a good time to treasure myself I was not deleting the game file

Reply:
Maybe "Discard changes"?

Reply:
I think Discard changes better describes what is happening than Cancel.